src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_handled_by_slr.cs

```diff
@@ -0,0 +1,151 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Config;
+    using NServiceBus.Pipeline;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_is_deferred_by_slr : NServiceBusAcceptanceTest
```

Comment:
We should change the name of the file as well. Currently it's `When_message_is_handeled_by_slr`